app/models/dependency.rb

```diff
@@ -33,34 +33,6 @@ def self.runtime_key(full_name)
     "rd:#{full_name}"
   end
 
-  # rails,rack,bundler
-  def self.for(gem_list)
-    versions = Redis.current.pipelined do
-      gem_list.each do |rubygem_name|
-        Redis.current.lrange(Rubygem.versions_key(rubygem_name), 0, -1)
-      end
-    end || []
-    versions.flatten!
-
-    return [] if versions.blank?
-
-    data = Redis.current.pipelined do
-      versions.each do |version|
-        Redis.current.hvals(Version.info_key(version))
-        Redis.current.lrange(Dependency.runtime_key(version), 0, -1)
-      end
-    end
-
-    data.in_groups_of(2).map do |(name, number, platform), deps|
-      {
-        name: name,
-        number: number,
-        platform: platform,
-        dependencies: deps.map { |dep| dep.split(" ", 2) }
-      }
-    end
-  end
-
```

Comment:
As we mentioned in another PR, it's used within api dependency controller, but now it's not used and api is routed to the bundler-api. https://github.com/rubygems/rubygems.org/commit/7e4c702b38f50aac32716401434c4864e886fa1f?diff=unified is the commit that removes ```Dependency.for``` with dependency controller.